src/test/java/org/folio/oaipmh/helpers/RepositoryConfigurationHelperTest.java

```diff
@@ -45,22 +45,17 @@
   @BeforeAll
   static void setUpOnce(Vertx vertx, VertxTestContext testContext) {
     okapiHeaders.put(OKAPI_TOKEN, "eyJhbGciOiJIUzI1NiJ9");
-    okapiHeaders.put(OKAPI_URL, "http://localhost:" + mockPort);
     OkapiMockServer okapiMockServer = new OkapiMockServer(vertx, mockPort);
     JsonObject conf = new JsonObject()
       .put("http.port", okapiPort);
     DeploymentOptions opt = new DeploymentOptions().setConfig(conf);
 
-    vertx.deployVerticle(RestVerticle.class.getName(), opt, testContext.succeeding(id ->
-      OaiPmhImpl.init(testContext.succeeding(success -> {
-        logger.info("mod-oai-pmh Test: setup done.");
-        okapiMockServer.start(testContext);
-      }))
-    ));
+    vertx.runOnContext(event -> testContext.verify(() -> okapiMockServer.start(testContext)));
```

Comment:
Is the verticle deployed automatically here?

Reply:
Actually the RestVerticle is not needed here but only MockServer. The RepositoryConfigurationHelper only sends requests to mod-configuration and returns result in Future. So I removed it to reduce unit test time run...

Reply:
ok, makes sense